extensions/panache/hibernate-orm-panache-kotlin/runtime/pom.xml

```diff
@@ -71,6 +71,22 @@
         </dependency>
     </dependencies>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>jcenter</id>
+            <name>JCenter</name>
+            <url>https://jcenter.bintray.com/</url>
+        </pluginRepository>
+        <pluginRepository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>bintray-kotlin-kotlin-dev</id>
```

Comment:
What do we need this repo for?

Reply:
for dokka

Reply:
But do we need both this and JCenter?

Reply:
no, actually.  I had added that to get at dev versions of dokka trying to fix a java 11 compat issue but since we're still building with java8, i can use a different version.

Reply:
gone!

Reply:
Thanks!